packages/library/index.js

```diff
@@ -35,9 +35,7 @@ module.exports = (neutrino, opts = {}) => {
         [require.resolve('@babel/preset-env'), {
           debug: neutrino.options.debug,
           useBuiltIns: 'entry',
-          targets: options.target === 'node' ?
-            { node: '8.10' } :
-            { browsers: 'ie 9' }
+          forceAllTransforms: true
```

Comment:
Should this be `false` when running in development?

Reply:
I don't *think* so, but there may be subtle bugs otherwise such as missing polyfills.